transport/polling/transport.go

```diff
@@ -10,14 +11,21 @@ import (
 
 // Transport is the transport of polling.
 type Transport struct {
-	Client *http.Client
+	Client      *http.Client
+	CheckOrigin func(r *http.Request) bool
 }
 
 // Default is the default transport.
 var Default = &Transport{
 	Client: &http.Client{
 		Timeout: time.Minute,
 	},
+	CheckOrigin: nil,
+}
+
+func Config(tsp *Transport) *Transport {
+	mergo.Merge(tsp, Default)
```

Comment:
Can we avoid the mergo dependency here? Since this struct only has 2 fields it seems like a large dependency to add just to avoid an if-statement?

It might be that merge does a recursive merge on the `Client` field? I still think it's overkill and that the caller just needs to be explicit if overriding that field  with the default value.

Reply:
I removed the dependency and now handle default fields directly in the methods which consume them.

Haha, you are right it was overkill. 😄 
I am actively using it in another project and got used to it so much that I didn't really overthink. 

Oh, and can you take a look at #86 ? Should be an easy merge aswell after my last commit and fixes problems for people with ARM arch.